packages/components/src/Tabs/TabPanel.tsx

```diff
@@ -10,9 +12,12 @@ export interface TabPanelProps extends AriaTabPanelProps {
 export const TabPanel = ({ state, ...props }: TabPanelProps) => {
   const ref = useRef(null);
   const { tabPanelProps } = useTabPanel(props, state, ref);
+  const selectedItemProps = state.selectedItem?.props;
+  const { classNames } = useTabContext();
```

Comment:
I think this we also don't need than, right @OsamaAbdellateef

Reply:
@sarahgm 
I think we need it, these `classNames` for styles via theme